resources/eventing/templates/pre-upgrade-job.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels: {{- include "eventing.preUpgrade.labels" . | nindent 4 }}
+  name: {{ template "eventing.fullname" . }}-remove-backendmodule
+  annotations:
+    helm.sh/hook-weight: "0"
+    helm.sh/hook: "pre-install,pre-upgrade"
+    helm.sh/hook-delete-policy: "before-hook-creation,hook-succeeded"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels: {{- include "eventing.preUpgrade.labels" . | nindent 4 }}
+  name: {{ template "eventing.fullname" . }}-remove-backendmodule
+  annotations:
+    helm.sh/hook-weight: "0"
+    helm.sh/hook: "pre-install,pre-upgrade"
+    helm.sh/hook-delete-policy: "before-hook-creation,hook-succeeded"
+rules:
+  - apiGroups: ["ui.kyma-project.io"]
+    resources: ["backendmodules"]
+    verbs: ["delete"]
```

Comment:
Why do we need this?

Reply:
The old `Backendmodule` had namespace annotation with `knative-eventing` which should be replaced with the new one with `kyma-system`. Hence it is needed.